codility_training/lessons.lesson01.Iterations.BinaryGap/sangmandu.py

```diff
@@ -0,0 +1,50 @@
+from random import randint
+
+def solution(n):
+    binary = []
+    while(n // 2 != 0):
+        if(n % 2 == 1):
+            binary.append(1)
+        else:
+            binary.append(0)
+        n = n // 2
+    binary.append(1)
+    
+    index = []
+    for x, y in enumerate(binary):
+        if(y == 1):
+            index.append(x)
+
+    max = 0
+    for i in range(len(index)-1):
+        temp = index[i+1] - index[i] - 1
+        if(max < temp):
+            max = temp
+
+    return max
+
+n = randint(1, 2147483647)
+solution(n)

```

Comment:
```suggestion
print(
    solution(32) == 0,
    solution(1041) == 5,
    solution(9) == 2,
)
```

이런식으로 직접 테스트 값과 결과 값을 비교해보는게 좋습니다!

Reply:
Codility 에서는 print하면 에러가 발생하긴 하는데, git에 올릴 때에는 따로 print 구문을 추가하라는 뜻일까요?